guides/common/modules/ref_managing-packages-on-the-base-operating-system.adoc

```diff
@@ -0,0 +1,55 @@
+[id="Managing_Packages_on_the_Base_Operating_System_{context}"]
+= Managing packages on the base operating system of {ProjectServer} or {SmartProxyServer}
+
+{Project} provides the `{foreman-maintain} packages` command to install and update packages on the base operating system of your {ProjectServer} or {SmartProxyServers}.
+
+[WARNING]
+====
+Installing and updating individual packages with `dnf` is not supported.
+While {Project} provides the `{foreman-maintain} packages unlock` command to bypass `{foreman-maintain} packages` and enable package management with `dnf`, this can lead to configuration inconsistencies and compromise the stability of the system.
+Always use `{foreman-maintain} packages` to manage packages.
```

Comment:
Does `manage` also mean `uninstall`? Because we don't offer a command for that :)

Reply:
Does that mean that if I install a package with foreman-maintain, then there is no supported way of uninstalling it? @evgeni

Reply:
It's complicated ;-)
The dnf plugin that we ship, which is responsible for this locking mechanism, doesn't affect removals, so people can do `dnf remove blah` just fine.
This might havoc their system in other ways, but that's not what the locking feature is trying to prevent.

Reply:
Thanks for explaining. I've changed this to "install and update".

I'm leaving "Managing" in the heading unchanged because that heading is already too long and I don't think it's such an issue there.